src/airflow/dags/test_DAG.py

```diff
@@ -0,0 +1,170 @@
+    },
+    # PICS credible sets from GWAS Catalog curated associations:
+    "gwas_catalog_curated_credible_set": {
+        "source_bucket": GWAS_CATALOG_BUCKET_NAME,
+        "source_object": "credible_set_datasets/gwas_catalog_curated",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/gwas_catalog_curated",
+    },
+    # PICS credible sets from GWAS Catalog summary statistics:
+    "gwas_catalog_sumstats_credible_set": {
+        "source_bucket": GWAS_CATALOG_BUCKET_NAME,
+        "source_object": "credible_set_datasets/gwas_catalog_summary_stats",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/gwas_catalog_summary_statistics",
+    },
+    # GWAS Catalog manifest files:
+    "gwas_catalog_manifests": {
+        "source_bucket": GWAS_CATALOG_BUCKET_NAME,
+        "source_object": "manifests",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/manifests",
+    },
+    # eQTL Catalog study index:
+    "eqtl_catalog_study_index": {
+        "source_bucket": EQTL_BUCKET_NAME,
+        "source_object": "study_index",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/study_index/eqtl_catalog",
+    },
+    # eQTL Catalog SuSiE credible sets:
+    "eqtl_catalog_susie_credible_set": {
+        "source_bucket": EQTL_BUCKET_NAME,
+        "source_object": "credible_set_datasets/susie",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/eqtl_catalog_susie",
+    },
+    # Finngen study index:
+    "finngen_study_index": {
+        "source_bucket": FINNGEN_BUCKET_NAME,
+        "source_object": f"{FINNGEN_RELEASE}/study_index",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/study_index/finngen",
+    },
+    # Finngen summary statistics:
+    "finngen_PICS_credible_set": {
+        "source_bucket": FINNGEN_BUCKET_NAME,
+        "source_object": f"{FINNGEN_RELEASE}/credible_set_datasets/finngen_pics",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/finngen_pics",
+    },
+    # Finngen SuSiE credible sets:
+    "finngen_susie_credible_set": {
+        "source_bucket": FINNGEN_BUCKET_NAME,
+        "source_object": f"{FINNGEN_RELEASE}/credible_set_datasets/finngen_susie_processed",
+        "destination_bucket": RELEASE_BUCKET_NAME,
+        "destination_object": f"releases/{RELEASE_VERSION}/credible_set/finngen_susie",
+    },
+    # L2G gold standard:
+    "gold_standard": {
+        "source_bucket": "genetics_etl_python_playground",
+        "source_object": "input/l2g/gold_standard/curation.json",
```

Comment:
This file is stored in Github, do you think it makes more sense loading it from remote?
Loading remote files is not super straightforward with Spark (SparkFiles solution doesnt work in Dataproc) but we do something similar here https://github.com/opentargets/gentropy/blob/06c611780e3ecb95b7d4b8b40a192dd8b984560b/src/gentropy/datasource/eqtl_catalogue/study_index.py#L46

Reply:
Fine with me,  I haven't change the logic of this part basically, we did copy from the other bucket. On the other hand, I do see some value to have this file included in the release folder.